data/sriharshamajeti.json

```
@@ -0,0 +1,29 @@
+{
+  "name": "Sara Jaoude",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "Lawyer turned **Digital Nomad**. Travel enthusiast Yoga & Peloton aficionado",
+  "avatar": "https://github.com/sriharshamajeti.png",
+  "links": [
+    {
+      "name": "Twitter",
+      "url": "https://twitter.com/sriharshamajeti",
+      "icon": "FaTwitter"
+    },
+    {
+      "name": "Instagram",
+      "url": "https://www.instagram.com/sree_harsha.majeti",
+      "icon": "FaInstagram"
+    },
+    {
+      "name": "LinkedIn",
+      "url": "https://www.linkedin.com/in/sri-harsha-majeti/",
+      "icon": "FaLinkedin"
+    },
+    {
+      "name": "GitHub",
+      "url": "https://github.com/sriharshamajeti",
+      "icon": "FaGitHub"
```

```suggestion
      "icon": "FaGithub"
```
The icon name casing is wrong